src/reference/antora/modules/ROOT/pages/mongodb.adoc

```diff
@@ -231,6 +231,22 @@ To configure that scenario, you can extend one message store bean from the other
     <int:priority-queue message-store="priorityStore"/>
 </int:channel>
 ----
+[[abstract-configurable-mongodb-message-store-with-auto-index-creation-disable]]
+=== Using AbstractConfigurableMongoDbMessageStore with auto index creation disable
```

Comment:
I don't think we need a dedicated section just for this feature.
Please, consider to move the content into the end of the `mongodb-message-store` section.